libs/agno/agno/vectordb/qdrant/qdrant.py

```diff
@@ -414,4 +523,4 @@ def optimize(self) -> None:
         pass
 
     def delete(self) -> bool:
-        return False
+        return self.client.delete_collection(collection_name=self.collection)
```

Comment:
The return type here shouldn't be `bool` then right?

Reply:
`delete_collection(...)` returns a `bool`.